tst/ProducerFunctionalityTest.cpp

```diff
@@ -95,7 +94,7 @@ TEST_F(ProducerFunctionalityTest, reset_stream_on_stream_error_with_frame_in_buf
 {
     STREAM_HANDLE streamHandle = INVALID_STREAM_HANDLE_VALUE;
     UINT32 i;
-    UINT32 totalFragments = 60;
+    UINT32 totalFragments = 10;
```

Comment:
Same concerns

Reply:
yes, the ones that I changed dont require to reach token rotation.